discord/client.py

```diff
@@ -846,7 +846,7 @@ def on_ready():
 
         Creates a :class:`Guild`.
 
-        Bot accounts generally are not allowed to create servers.
+        Bot accounts over 10 total guilds are typically not allowed to create guilds.
```

Comment:
bot accounts *in* over 10 total guilds

Reply:
Better wording:

"Bot accounts in over 10 guilds are not allowed to create guilds."